src/pkg/sealedSecrets/SealedSecretsWrapper.go

```diff
@@ -67,3 +75,41 @@ func prefix(prefix string, keys ...string) []string {
 	}
 	return res
 }
+
+func writeSecretAndValuesToFile(secret []byte, keys ...string) {
+	secretName, _ := utils.FindValue(secret, "metadata.name")
+	writeSecretToFile(secret, secretName)
+	writeValuesToFile(secret, secretName, keys...)
+}
+
+func writeSecretToFile(secret []byte, secretName any) {
+	filenameSecret := fmt.Sprintf("%s/%s.yaml", utils.DEBUG_DIR, secretName)
+	err := ioutil.WriteFile(filenameSecret, secret, 0644)
+	if err != nil {
+		log.Printf("WARNING: Failed to write %s.", secretName)
+	}
+}
+
+func writeValuesToFile(secret []byte, secretName any, keys ...string) {
+	prefixed := prefix("data:", keys...)
+	values, _ := utils.FindValues(secret, prefixed...)
+	filenameValues := fmt.Sprintf("%s/%s-values.txt", utils.DEBUG_DIR, secretName)
+	fileValues, err := os.Create(filenameValues)
+	if err != nil {
+		log.Printf("WARNING: Failed to write %s.", filenameValues)
+		return
+	}
+	defer fileValues.Close()
+
+	for key, val := range values {
+		decoded, err := utils.Base64Decode(val)
```

Comment:
Top! Während dem Lesen des Codes wollt ich schon fragen, ob wir die Values nicht vielleicht gleich Base64 decoden sollen, damit man das nicht mehr manuell machen muss. 

Eventuell noch ein Kommentar in die Secrets reinschreiben, dass sie so nicht direkt in OpenShift eingespielt werden können, weil sie ja nicht encoded sein. Aber da bin ich mir selbst nicht ganz sicher.